app/src/main/java/org/wikipedia/settings/SwitchPreferenceMultiLine.kt

```diff
@@ -0,0 +1,21 @@
+package org.wikipedia.settings
+
+import android.R
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.TextView
+import androidx.preference.PreferenceViewHolder
+import androidx.preference.SwitchPreferenceCompat
+
+open class SwitchPreferenceMultiLine : SwitchPreferenceCompat {
+    constructor(ctx: Context?, attrs: AttributeSet?, defStyle: Int) : super(ctx, attrs, defStyle)
+    constructor(ctx: Context?, attrs: AttributeSet?) : super(ctx, attrs)
+    constructor(ctx: Context?) : super(ctx)
+
+    override fun onBindViewHolder(holder: PreferenceViewHolder) {
+        super.onBindViewHolder(holder)
+        holder.itemView.findViewById<TextView>(R.id.title)?.also { textView ->
+            textView.isSingleLine = false
```

Comment:
Couldn't this just be:
`holder.itemView.findViewById<TextView>(R.id.title).isSingleLine = true`


Also, the identifier `R.id.title` should be fully qualified. (Don't import `android.R`)